backend/src/routes/api/notebooks/index.ts

```diff
@@ -83,9 +83,13 @@ module.exports = async (fastify: KubeFastifyInstance) => {
         }>,
       ) => {
         const { namespace, name } = request.params;
-        const data = await sanitizeNotebookForSecurity(fastify, request, request.body);
+        const data = (await sanitizeNotebookForSecurity(
+          fastify,
+          request,
+          request.body,
+        )) as Notebook;
```

Comment:
```suggestion
        const data = await sanitizeNotebookForSecurity<Notebook>(
          fastify,
          request,
          request.body,
        );
```

I had to make it the weaker of the two options -- but it is a generic and you can just say "I have a full notebook"